submissions/xT0PERx/Friends app/js/index.js

```diff
@@ -0,0 +1,177 @@
+const allFriends = {
+  currentList: [],
+  changeList: [],
+  numberOfFriends: 72,
+};
+
+const REQUEST_URL = "https://randomuser.me/api/1.3/";
+const ALL_USERS_CARDS_FILTERS = allFriends.changeList;
+const ALL_USERS_CARDS = allFriends.currentList;
+const USERS_LIST = document.getElementById("usersList");
+const FILTERS_MENU = document.getElementById("filters_menu");
+const ABC_BUTTON_FILTER = document.getElementById("Abc");
+const ZYX_BUTTON_FILTER = document.getElementById("Zyx");
+const MALE_BUTTON_FILTER = document.getElementById("male");
+const FEMALE_BUTTON_FILTER = document.getElementById("female");
+const YOUNGER_FILTER_BUTTON = document.getElementById("younger-ftr");
+const SENIOR_FILTER_BUTTON = document.getElementById("senior-ftr");
+const RESET_BUTTON = document.getElementById("reset");
```

Comment:
Do I understand this point of the comment correctly?
___When you need to decide which sort condition were selected, prefer value of input to “check class name of pressed element”, or “compare pressed element to saved element” etc. You can have value “female”, get value and then just use it___
It is better to assign a __value__ to the __button__ element and use it in my code than to assign to the element __id__?

Reply:
> value of input

vs 

> value to the button

:)

And WDYM under?

>  to assign to the element id

`value` will allow you to understand what element were clicked

Reply:
> value of input

vs 

> value to the button

:)

And WDYM under?

>  to assign to the element id

`value` will allow you to understand what element was clicked

Reply:
To understand which element was clicked, I use  __\<button id="name-ascend"__. My question is as follows. For elements which, according to the specification, provide such properties as __"value"__ it is better to use them, and for elements whitch dont have __"value"__  - i use __id__(if I want to apply to them JS code). 
I.e __\<button value="name-ascend"___ instead __\<button id="name-ascend"__. 
I seem to have caught up, but I still want to hear an expert's opinion :)

Reply:
> My question is as follows. For elements which, according to the specification, provide such properties as "value" it is better to use them

Yes.

> and for elements whitch dont have "value" - i use id(if I want to apply to them JS code).

No. `id` does not replace `value`. It is just a unique identificator and it can be used here and there. Generally speaking, it has it's own "do's and don'ts".

You don't need `button`, you need `radio` because you have two buttons and both buttons can have two states - checked and unchecked. It is a list of radio buttons.

So what you should do is replace `button` with `input type='radio'` and then get the value of input from the `change` event.